examples/animation-generic-logo/index.html

```diff
@@ -41,20 +41,20 @@
 
         <a-entity id="dots1" rotation="0 0 45">
           <a-animation attribute="scale" from="1 0 1" to="1 0.6 1" begin="800" dur="750" fill="both" easing="ease-out"></a-animation>
-          <a-cylinder position="0 2 0" radius="0.04" height="2" translate="0 -1 0" color="#EF2D5E" open-ended="true" side="double">
```

Comment:
I haven't tried but removing translation should change the animation of this demo. Does this look all right?